nvm.sh

```diff
@@ -1231,6 +1231,25 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ -z "$NVM_CPU_THREADS" ] || [[ ! $NVM_CPU_THREADS =~ ^[0-9]+$ ]] ; then
```

Comment:
this syntax is bash-specific - `nvm` must work on all POSIX systems, including ksh, dash, sh, bash, and zsh.